From 9060da2746827c1d0ee0e308bed087156254882b Mon Sep 17 00:00:00 2001
From: Danny Nold <dannynold@freescale.com>
Date: Fri, 14 Oct 2011 12:50:39 -0500
Subject: ENGR00160061 - Mach-MX6: Optimize HDMI clock management

- Decouple HDMI IAHB clock from HDMI ISFR clock, in order to
allow IAHB clock to be disabled while keeping ISFR clock enabled.

Signed-off-by: Danny Nold <dannynold@freescale.com>
---
 arch/arm/mach-mx6/clock.c |    1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm/mach-mx6/clock.c b/arch/arm/mach-mx6/clock.c
index 9e76568..9cc316f 100644
--- a/arch/arm/mach-mx6/clock.c
+++ b/arch/arm/mach-mx6/clock.c
@@ -3482,7 +3482,6 @@ static struct clk hdmi_clk[] = {
 	 __INIT_CLK_DEBUG(hdmi_isfr_clk)
 	.id = 0,
 	.parent = &pll3_pfd_540M,
-	.secondary = &hdmi_clk[1],
 	.enable_reg = MXC_CCM_CCGR2,
 	.enable_shift = MXC_CCM_CCGRx_CG2_OFFSET,
 	.enable = _clk_enable,
-- 
1.7.9.5

